arches/app/utils/data_management/resources/formats/rdffile.py

```diff
@@ -890,9 +890,9 @@ def find_matching_branch(self, k, v, tree_node, result, tile=None, indent=0):
                     try:
                         self.printline("Found multiple matches!", indent)
                         # if this doesn't throw an error then keep the possible branch "p"
-                        for k, v in vi.items():
+                        for k1, v in vi.items():
                             matched_branch = self.find_matching_branch(
-                                k, v, p[0], {}, tile, indent + 1
+                                k1, v, p[0], {}, tile, indent + 1
```

Comment:
If there is a scoping problem with k it seems it could potentially be a problem with v as well. I think a safer and maybe clearer solution would be `for nested_key, nested_value in vi.items() ... nested_key, nested_value, p[0], {}, tile, indent + 1`